akka-remote/src/test/scala/akka/remote/artery/ArteryMultiNodeSpec.scala

```diff
@@ -58,7 +58,7 @@ abstract class ArteryMultiNodeSpec(config: Config)
       setup: Option[ActorSystemSetup] = None): ActorSystem = {
     val config =
       extraConfig.fold(localSystem.settings.config)(str =>
-        ConfigFactory.parseString(str).withFallback(localSystem.settings.config))
+        ConfigFactory.parseString(str).resolve().withFallback(localSystem.settings.config))
```

Comment:
is this still needed if we remove the `${...config-ssl-engine}` from reference conf, or what was the reason?

Reply:
Tests in `RotatingKeysSSLEngineProviderSpec` were also using substitution. I've rewritten them to avoid it.